app/controllers/welcome_controller.rb

```diff
@@ -1,5 +1,5 @@
 class WelcomeController < ApplicationController
   def index
-    @categories = Dataset.chosen_categories
+    @categories = Dataset.chosen_categories.shuffle
```

Comment:
Why?

Reply:
eh, why not? it's the first page, make it random. draws attention to it. @kaars what do you think?

Reply:
I think this is fine for now, but as the number of categories increases it's gonna be a mess. This should be a word cloud, like this for example: https://github.com/zverok/magic_cloud

Obviously, it doesn't have to happen right now but eventually it'd be nice.